From: Kill Your TV <killyourtv@i2pmail.org>
Date: Sat, 11 Oct 2014 00:12:40 +0000
Subject: write proper pidfile

Signed-off-by: Kill Your TV <killyourtv@i2pmail.org>
---
 other/bootstrap_daemon/tox-bootstrapd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/other/bootstrap_daemon/tox-bootstrapd.c
+++ b/other/bootstrap_daemon/tox-bootstrapd.c
@@ -659,7 +659,7 @@
     const pid_t pid = fork();
 
     if (pid > 0) {
-        fprintf(pidf, "%d", pid);
+        fprintf(pidf, "%d\n", pid);
         fclose(pidf);
         syslog(LOG_DEBUG, "Forked successfully: PID: %d.\n", pid);
         return 0;
